pkg/drivers/kvm/kvm.go

```diff
@@ -346,22 +383,22 @@ func (d *Driver) Create() (err error) {
 	}
 
 	store := d.ResolveStorePath(".")
-	log.Infof("Setting up store path in %s ...", store)
+	log.Infof("setting up store path in %s ...", store)
 	// 0755 because it must be accessible by libvirt/qemu across a variety of configs
 	if err := os.MkdirAll(store, 0755); err != nil {
 		return errors.Wrap(err, "creating store")
 	}
 
-	log.Infof("Building disk image from %s", d.Boot2DockerURL)
-	if err = pkgdrivers.MakeDiskImage(d.BaseDriver, d.Boot2DockerURL, d.DiskSize); err != nil {
-		return errors.Wrap(err, "error creating disk")
+	log.Infof("building disk image from %s", d.Boot2DockerURL)
+	if err := pkgdrivers.MakeDiskImage(d.BaseDriver, d.Boot2DockerURL, d.DiskSize); err != nil {
+		return errors.Wrap(err, "creating disk")
 	}
 
 	if d.ExtraDisks > 20 {
 		// Limiting the number of disks to 20 arbitrarily. If more disks are
 		// needed, the logical name generation has to changed to create them if
 		// the form hdaa, hdab, etc
-		return errors.Wrap(err, "cannot create more than 20 extra disks")
+		return errors.New("creating more than 20 extra disks")
```

Comment:
The before message is a better user facing error, `cannot create` vs `creating`, although we should have validation for this way before

Reply:
agree, reworded it